sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -159,7 +162,8 @@ export class ApplicationStudentsController extends BaseController {
       "Selected offering id is invalid or " +
       "invalid study dates or selected study start date is not within the program year" +
       "or APPLICATION_NOT_VALID or INVALID_OPERATION_IN_THE_CURRENT_STATUS or ASSESSMENT_INVALID_OPERATION_IN_THE_CURRENT_STATE " +
-      "or INSTITUTION_LOCATION_NOT_VALID or OFFERING_NOT_VALID.",
+      "or INSTITUTION_LOCATION_NOT_VALID or OFFERING_NOT_VALID." +
```

Comment:
Please remove the period at the end.
Just a point for the future because it did not start in this PR, the idea of the comments is to be more friendly than just listing the error codes. They can potentially be consumed by an audience external to this Team where the API descriptions would be helpful.

Reply:
This is how the concatenated string will look like in Swager:
"or INSTITUTION_LOCATION_NOT_VALID or OFFERING_NOT_VALIDor Invalid program or offering intensity"
Please add a space and a period as below

```ts
 "or INSTITUTION_LOCATION_NOT_VALID or OFFERING_NOT_VALID " +
 "or invalid program or offering intensity.",
```